cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/AuditMetadataStorage.java

```diff
@@ -154,6 +158,26 @@ private void emitMetrics(String metricSuffix) {
     metricsCollector.increment(Constants.Metrics.MetadataStorage.METRICS_PREFIX + metricSuffix, 1L);
   }
 
+  private void emitMutationsMetrics(MetadataMutation mutation, String suffix) {
+    switch (mutation.getType()) {
+      case DROP:
```

Comment:
isn't this equivalent to: emitMetrics(mutation.getType().name().toLowerCase() + suffix ?

Reply:
Also I am not sure if it is good to compute this string every time. I would rather have constants defined. Perhaps you can map mutation type to a constant?